src/Core/Components/List/ListComponentBase.razor.cs

```diff
@@ -550,67 +571,20 @@ protected virtual async Task RaiseChangedEventsAsync()
                 builder.AddContent(0, ChildContent);
             });
         }
-
-        EventCallback<string> OnSelectCallback(TOption? item)
-        {
-            return EventCallback.Factory.Create<string>(this, (e) => OnSelectedItemChangedHandlerAsync(item));
-        }
-    }
-
-    /// <summary />
-    protected virtual async Task OnChangedHandlerAsync(ChangeEventArgs e)
-    {
-        if (e.Value is not null && Items is not null && !Multiple)
-        {
-            TOption? item = Items.FirstOrDefault(i => GetOptionValue(i) == e.Value.ToString());
-
-            await OnSelectedItemChangedHandlerAsync(item);
-        }
-        if (e.Value is not null && Multiple)
-        {
-            InternalValue = e.Value.ToString();
-        }
     }
 
     protected virtual async Task OnKeydownHandlerAsync(KeyboardEventArgs e)
     {
-        if (e is null || Multiple || Items is null)
+        if (e is null || Multiple)
         {
             return;
         }
 
+        string id = await Module!.InvokeAsync<string>("getAriaActiveDescendant", Id);
```

Comment:
Could you test the value of Module: `if (e is null || Multiple || Module is null)`.
If the compiler says that Module may be null, it's best to reject this case explicitly (and avoid using `!`).